doc/src/guides/solving/solve-equation-algebraically.md

```diff
@@ -0,0 +1,90 @@
+# Solve an equation algebraically
+
+Use SymPy to solve an equation algebraically. For example, solving $x^2 = 4$ yields $x \in \{-2,2\}$.
```

Comment:
This is not a good example because it would be better done in other ways e.g. by using a function specifically for finding polynomial roots.

Also this and many of the other examples give the impression that `solve` is for finding numeric solutions. It would be better to at least show an example that gives non-integer results like:
```python
In [4]: solve(x**2 + x - 1, x)
Out[4]: 
⎡  1   √5    √5   1⎤
⎢- ─ + ──, - ── - ─⎥
⎣  2   2     2    2⎦
```
Ideally you should emphasise the fact that we can solve equations involving symbols:
```python
In [5]: solve(x**2 - y, x)
Out[5]: [-√y, √y]
```
That's what SymPy's solve function can do that other functions like scipy's `fsolve` cannot do.